releases/development/master/extra/install-chef-suse.sh

```diff
@@ -797,6 +822,7 @@ fi
 # in cookbook dependencies)
 #
 for i in crowbar deployer dns ipmi logging network ntp provisioner pacemaker \
+         suse-enterprise-storage \
```

Comment:
Actually, I don't think it will break, because the other barclamps in that list (database, rabbitmq, openstack, keystone etc.) don't exist on my test system, and yet the install script runs fine. I just end up with a whole bunch of stuff like `/opt/dell/barclamps//database is not a barclamp, ignoring.` in `/var/log/crowbar/install.log`. I'd expect the same on a cloud deploy where the suse-enterprise-storage barclamp is not present.